src/Native/pkg/runtime.native.System.Net.Security/runtime.native.System.Net.Security.builds

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="runtime.native.System.Net.Security.pkgproj"/>
+    <Project Include="centos\runtime.native.System.Net.Security.pkgproj">
+      <!-- currently we build both "centos" and "ubuntu" packages on Linux,
+           but will only publish one of the packages based on what OS we
+           were actually building on.  
+           Remove this when fixing https://github.com/dotnet/corefx/issues/5190 -->
+      <OSGroup>Linux</OSGroup>
+      <Platform>amd64</Platform>
```

Comment:
You shouldn't need to specify the "Platform" property, the pkgproj files already restrict it to amd64 builds.